src/alpinelinux.ipxe

```diff
@@ -0,0 +1,29 @@
+#!ipxe
+
+# Alpine Linux netboot.xyz ipxe script.
```

Comment:
Can just leave this as Alpine Linux to make it consistent with the other iPXE files.